src/lib/network/clients/save_network.rb

```diff
@@ -346,14 +346,14 @@ def configure_target
 
     # Sets default network service
     def set_network_service
-      if Mode.autoinst
-        NetworkAutoYast.instance.set_network_service
-        return
+      if Mode.autoinst && !Yast::Lan.autoinst.managed.nil?
```

Comment:
I would expect these lines to be near the place where the network service is actually set. Is the condition used elsewhere?

Reply:
I agree, get rid of it